meta-mender-tegra/meta-mender-tegra-jetpack5/README.md

```diff
@@ -0,0 +1,55 @@
+# meta-mender-tegra
+
+Mender integration layer for NVIDIA Tegra hardware.
+
+The supported and tested boards are:
+
+- AGX Orin
+- AGX Xavier
+- Orin Nano
+
+Visit the individual board links above for more information on status of the
```

Comment:
Contrary to the readme for jetpack 4, this readme does not add links for the boards. So it's probably best to remove this sentence to not cause confusion.